guides/common/modules/proc_removing-an-scc-account.adoc

```diff
@@ -0,0 +1,30 @@
+[id="Removing_an_SCC_Account_from_{project-context}_{context}"]
+= Removing an SCC Account from {Project}
+
+Use the following procedure to remove your SCC account from {Project}.
+To use the CLI instead of the {ProjectWebUI}, see the xref:cli-Removing_an_SCC_Account_from_{project-context}_{context}[].
+
+[WARNING]
+====
+If you want to switch SCC accounts and retain your synchronized content, do not delete your old SCC account, even if it is expired.
+Instead, change the login and password of your SCC account.
+If you delete your old SCC account, you cannot reuse existing repositories, products, content views, and composite content views.
+For more information, see xref:Switching_SCC_Accounts_{context}[].
+====
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *SUSE Subscriptions*.
+. For your SCC account, click *Delete* in the _actions_ menu.
+. Click *Confirm* to remove your SCC account from {Project}.
+
+[id="cli-Removing_an_SCC_Account_from_{project-context}_{context}"]
+.CLI procedure
+* Remove your SCC account from {Project}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer scc_manager scc_accounts delete \
+--id _My_SCC_Account_ID_ \
+--location-id _My_Location_ID_ \
```

Comment:
As the scc_account is bound to a location/organization already and the scc_account_id is set, location / organization is not necessary to be set. These options will be removed soon-ish.

Reply:
Applied.